src/App.js

```diff
@@ -7,9 +7,11 @@ import {
   Route,
   Link,
   useRouteMatch,
+  Redirect,
 } from 'react-router-dom';
```

Comment:
What do you think about adding a space after this line? So libraries and what is created by you are separated; it's something I use to do to keep code cleaner and more organized :D